fluent-bit-init-process_test.go

```diff
@@ -0,0 +1,295 @@
+package main
+
```

Comment:
for name, let' do all `-` or `_` to keep consistent

Reply:
Mm-hmm, it's because Go unit test file must end with `_test.go`

Reply:
I see. Then I would prefer all `_`